README.md

```diff
@@ -1 +1,72 @@
-# Leaderboard-app
\ No newline at end of file
+# Leaderboard-app
+
+> This project is about a leadboard app, in which, folks can add player names with their scores.
+
+## Live demo
+
+<!-- The website was build using Github Pages: [Live Version](https://safar1212.github.io/Todo-list-software/) -->
+
+## Built With
+
+-Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more).
+-Webhint (A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors).
+-Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+
+### Prerequisites
+
+## VScode or other editor
+
+### Install
+
+#### Cloning the project
+
+
+
+git clone [https://safar1212.github.io/Todo-list-software/](https://safar1212.github.io/Todo-list-software/) <Your-Build-Directory>
+
+## Getting packages and debuging with Stylelint
+
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+##### For validation detection using Stylelint Run
+
+npx stylelint "\*_/_.{css,scss}"
+
+##### from parent source directory
+
+## Getting packages and debuging with Webhint
+
+npm init -y
+npm install --save-dev hint@6.x
+
+##### For validation detection using Webhint Run
+
+npx hint .
+
+## Authors
+
+👤 **Safar Ali**
+
+- GitHub: [@Safar Ali](https://github.com/safar1212)
+- LinkedIn: [Safar Ali](https://www.linkedin.com/in/safar-ali999/)
+- Twitter: [Safar Ali](https://twitter.com/SafarAli999)
+ 
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/safar1212/Todo-list-software/issues).
```

Comment:
- [ ] The issue link navigates to the `Todo-list-software` issues page. Please change the link so that it takes the user to the current project issues page.